.github/workflows/clang-tidy.yml

```
@@ -0,0 +1,29 @@
+name: clang-tidy
```

This currently names both the workflow and the job as `clang-tidy`. What if we rename the workflow to have a nicer PR CI checklist:

```suggestion
name: Code linting
```